src/main/java/Duke/Duke.java

```diff
@@ -0,0 +1,151 @@
+package Duke;
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+
+public class Duke {
+    public static final String Line = "    ____________________________________________________________";
+    private static ArrayList<Task> tasks = new ArrayList<>();
+    private static int taskSum = 0;
+    public static void printList() {
+        Task now;
+        System.out.println(Line);
+        System.out.println("     Here are the tasks in your list:");
+        for(int i=0; i<taskSum; i++){
```

Comment:
There should be a spacing between the brackets _(_ and _{_ in the **for** statement. The same concept applies to the rest of the code.